core-ui/src/state/navigation/fetchPermissions.ts

```diff
@@ -0,0 +1,24 @@
+import { GetRecoilValue } from 'recoil';
+import { activeNamespaceIdState } from '../activeNamespaceIdAtom';
+import { getPostFn } from '../utils/getPostFn';
+
+export const fetchPermissions = async (get: GetRecoilValue) => {
+  const activeNamespaceId = get(activeNamespaceIdState);
+  const postFn = getPostFn(get);
+
+  if (postFn) {
+    const namespaceName = activeNamespaceId ? activeNamespaceId : '*';
+    const path = '/apis/authorization.k8s.io/v1/selfsubjectrulesreviews';
+    const ssrr = {
+      typeMeta: {
+        kind: 'SelfSubjectRulesReview',
+        aPIVersion: 'authorization.k8s.io/v1',
```

Comment:
My mistake from made 19 months ago... 🤦 
```suggestion
        APIVersion: 'authorization.k8s.io/v1',
```